backend/src/main/java/com/empire/Noble.java

```diff
@@ -21,11 +22,30 @@ static Noble makeNoble(Culture culture, int date) {
 		return n;
 	}
 
+	double calcLevel() {
+		return Math.sqrt(experience + 1);
+	}
+
+	double calcPlantMod() {
+		return calcLevel() * Constants.noblePlantModPerLevel;
+	}
+
+	double calcRecruitMod() {
+		return calcLevel() * Constants.nobleRecruitModPerLevel;
+	}
+
+	double calcTaxMod() {
+		return calcLevel() * Constants.nobleTaxModPerLevel;
+	}
+
+	void addExperience() {
+		experience++;
```

Comment:
You might consider making the rate of experience increase a constant, or add a scaling factor on it in calcLevel().

Reply:
At the moment, I'm leaning toward not introducing identity-valued constants. The thinking is that if this is changed, the constant can be added at that time in a backward-compatible way.